src/tests/ftest/control/version.py

```diff
@@ -22,32 +22,69 @@ class DAOSVersion(TestWithServers):
     def __init__(self, *args, **kwargs):
         """Initialize a DAOSVersion object."""
         super().__init__(*args, **kwargs)
+
         # Don't waste time starting servers and agents.
         self.setup_start_servers = False
         self.setup_start_agents = False
 
+        self.errors = []
+
+    def report_result(self):
```

Comment:
The `tearDown()` and `setUp()` methods provide markers to help find the beginning and end of the test - a sequence of 80 `=` characters.  We typically just end each `test_*` method with `self.log.info("Test passed")` to indicate test passing (in addition to avocado's own messaging) and rely on `self.fail()` or assertions to indicate the test failure before calling `tearDown()`.  If we need more than this it should be standardized so we can use the same search string for all test job.log results.   That said, it can be addressed in a different PR.

Reply:
In fact, I have done it this way, to keep the same behavior as it was originally done in the PR.
However, if there is some standard to respect I will fix this in a follow up PR as this one has been already merged.

- [ ] Use standardized output message

Reply:
Fixed with PR #12615